### PR TITLE
[IMP] Changed the name(OBI) and ref(Reference) fields from statement …

### PR DESCRIPTION
**Pull request by sadamo**
…line to be dynamically imported from the ofx file.

This change was made because the .ofx files from Brazil where diferents. 
This way, the module becomes more flexible.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Interesting change, but you should provide a default value for the fields that guarantees that existing profiles don't break and newcomers to this module don't lose in the technical part of the OFX file.

File: account_statement_ofx_import/statement.py [hunk 1]
```diff
@@ -19,12 +19,23 @@
...
 class AccountStatementProfil(orm.Model):
     _inherit = "account.statement.profile"
 
+    _columns = {
+        'ofx_parser_label': fields.char('OFX parser Label field',
+                                        required=True),
+        'ofx_parser_ref': fields.char('OFX parser Ref field', required=True),
+    }
+
+    _defaults = {
```
Comment (sadamo) [inline account_statement_ofx_import/statement.py:34]: @pedrobaeza
The original values for the fields, are setted as defaults here.

File: account_statement_ofx_import/parser/ofx_parser.py [hunk 2]
```diff
@@ -61,8 +61,8 @@ def _parse(self, *args, **kwargs):
             res.append({
                 'date': transaction.date,
                 'amount': transaction.amount,
-                'ref': transaction.type,
-                'label': transaction.payee,
+                'ref': getattr(transaction, self.profile.ofx_parser_ref),
```
Comment (pedrobaeza) [inline account_statement_ofx_import/parser/ofx_parser.py]: You should protect this in case of the provided label is incorrect.

**Comment by pedrobaeza** [comment]
Ah, sorry, I suppose I was expecting new API and then didn't see the old defaults section.

**Comment by pedrobaeza** [comment]
I have added the account-financial-tools repo to runbot to get a green result in next commit.

**Comment by pedrobaeza** [comment]
This is still in red. @sadamo, are you still interested in this change? If so, please rebase and check Travis status.

**Comment by pedrobaeza** [comment]
Closing as no answer.

**Pull request closed without merging**